test/pytest/test_device_diagnostics.py

```diff
@@ -10,18 +10,17 @@
 @pytest.mark.usefixtures("phone_unlocked")
 def test_device_list_logs(harness):
     ret = GetDiagnosticFilesList(DiagnosticsFileList.LOGS).run(harness)
+    try:
+        if not ret.files:
+            raise RuntimeError("No log files")
 
-    if not ret.files:
-        log.info("No log files")
-        assert True
-        return
+    except RuntimeError as err:
+        log.error(err)
+        pytest.exit(err)
```

Comment:
this seems excessive. First call runtime error, to just catch it in next line? :) 

just do not catch the error and pytest will exit with full trace.

at best create own error after RuntimeError - i.e. 
```
class DiagnosticsError(RuntimeError)
    pass
```

Reply:
Well, my intention was to avoid abrupt termination and just log the error and exit.

Reply:
Changed as requested.